tests/src/Interop/COM/NETClients/IDispatch/Program.cs

```diff
@@ -151,6 +152,23 @@ static void Validate_StructNotSupported()
             Assert.Throws<NotSupportedException>(() => dispatchTesting.DoubleHVAValues(ref input));
         }
 
+        static void Validate_LCID_Marshaled()
+        {
+            var dispatchTesting = (DispatchTesting)new DispatchTestingClass();
+            CultureInfo oldCulture = CultureInfo.CurrentCulture;
+            CultureInfo newCulture = new CultureInfo("es-ES", false);
+            try
+            {
+                CultureInfo.CurrentCulture = newCulture;
+                int lcid = dispatchTesting.PassThroughLCID();
+                Assert.AreEqual(CultureInfo.CurrentCulture.LCID, lcid);  // CLR->Dispatch LCID marshalling is explicitly hardcoded to en-US instead of passing the current culture.
```

Comment:
I don't understand the comment this case. We are verifying all of this against `newCulture` which is `es-ES`?

Reply:
I forgot to change the assert to verify against `englishCulture` when I moved this over. That's why the tests are failing.